backend/src/helpers/token/create-token/create-token.helper.ts

```diff
@@ -0,0 +1,9 @@
+import jwt from 'jsonwebtoken';
+import { JWT_CONFIG } from '../../../../config/jwt.config';
```

Comment:
```suggestion
import { JWT_CONFIG } from '~/config/jwt.config';
```
`~` is an alias for the `root` in the *backend*